nikola/data/themes/base/templates/post_helper.tmpl

```diff
@@ -41,7 +41,7 @@
 
 <%def name="open_graph_metadata(post)">
 %if use_open_graph:
-    <meta property="og:site_name" content="${blog_title|striphtml}">
+    <meta property="og:site_name" content="${blog_title|striphtml,h}">
```

Comment:
Yes. It is valid syntax. http://docs.makotemplates.org/en/latest/filtering.html
striphtml call markupsafe.Markup.striptags().
https://github.com/mitsuhiko/markupsafe/blob/master/markupsafe/__init__.py#L148
This method only remove <!--.*?-->|<[^>]*> blocks and unescape.